bin/nvidia-container-toolkit

```diff
@@ -2,12 +2,38 @@
 
 set -eu
 
-# Ensure nvidia support setup correctly #
+device_wait() {
```

Comment:
A bit sad we need to do all of this, because we cannot simply be `After=snap.nvidia-assemble.service` (or some such)